Formula/minetest.rb

```diff
@@ -3,12 +3,12 @@ class Minetest < Formula
   homepage "https://www.minetest.net/"
 
   stable do
-    url "https://github.com/minetest/minetest/archive/5.0.1.tar.gz"
-    sha256 "aa771cf178ad1b436d5723e5d6dd24e42b5d56f1cfe9c930f6426b7f24bb1635"
+    url "https://github.com/johnlockard/minetest/archive/5.1.0.tar.gz"
```

Comment:
This doesn't look official. Why aren't you using the official repository?

If there's a particular patch that's required, you can do that separately in a `patch do` block, assuming the patch has been accepted upstream:

```ruby
patch do
  url "https://github.com/org/repo/commit/whatever.patch?full_index=1"
  sha256 "abcdef"
end
```

Very similar to the `resource do` block.